annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastFieldElement.java

```diff
@@ -0,0 +1,56 @@
+package com.exactpro.epfast.annotation.processing;
+
+import com.exactpro.epfast.annotations.FastField;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.type.TypeMirror;
+
+public class FastFieldElement {
+    private ExecutableElement setterMethod;
+
+    private String fieldName;
+
+    public FastFieldElement(Element setterMethod) {
```

Comment:
It's better to pass parameter as `ExecutableElement`. Current constructor signature doesn't tell a developer that we assume that we expect `setterMethod` to be `ExecutableElement`